assets/src/edit-story/components/library/panes/media/media3p/media3pPane.js

```diff
@@ -47,12 +48,18 @@ import paneId from './paneId';
 import ProviderTab from './providerTab';
 
 const ProviderTabSection = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
   margin-top: 16px;
   margin-bottom: 16px;
   padding: 0 24px;
 `;
 
-const MediaSubheading = styled.div`
+const MediaSubheading = styled(Text).attrs(() => ({
```

Comment:
just curious why as a `div`?

Reply:
Yeah this should be a proper heading for a11y reasons.

On the Shapes panel we use a h2 for the overall heading:

<img width="445" alt="Screenshot 2021-03-31 at 10 48 40" src="https://user-images.githubusercontent.com/841956/113117543-d83ce900-920e-11eb-909f-9b929db95b7c.png">

So here it could be a h2 as well, assuming that the overall region is announced properly due to the tabpanel.

Otherwise it should be a h3, with a visually hidden h2 at the very top for "Explore Media"

Reply:
Yea I was just retaining what it originally was, not much thought behind it. Awesome! will update

Reply:
(thanks, sorry for the scope creep)